BoxContentSDK/BoxContentSDK/Requests/BOXRecentItemsRequest.h

```diff
@@ -0,0 +1,41 @@
+//
+//  BOXRecentItemsRequest.h
+//  Pods
+//
+//  Created by Andrew Dempsey on 12/19/16.
+//
+//
+
+#import <BoxContentSDK/BoxContentSDK.h>
+
+@interface BOXRecentItemsRequest : BOXRequest
+
+/**
+ Number of records to fetch. 
+ - Minimum allowed value: 0
+ - Maximum allowed value: 1000
+ - Defaults to 100
```

Comment:
Is this the API default? The SDK should not impose any value, simply let the API default come through (and if that changes server-side, the SDK would automatically serve the new default).